server/server.js

```diff
@@ -1,35 +1,23 @@
-var dvv = require('./dvv.js');
+var imageUtils = require('./utils/image.js');
+var utils = require('./utils/utils.js');
 
-//Function to create arrays for testing purposes
-var createMatrixArrays = function(matrixSize, arrayLength){
-  var randomArray = function(n){
-    var result = [];
-    for(var i = 0; i < n ; i++){
-      var row = [];
-      for (var j = 0; j < n; j++){
-        row.push(Math.floor(Math.random()*100));
-      }
-      result.push(row);
-    }
-    return result;
-  };
+var init = function(dataArray, width, height){
 
-  var result = [];
-  for(var i = 0; i < arrayLength; i++){
-    result.push(randomArray(matrixSize));
-  }
-  return result;
-};
+  var dvv = require('./dvv.js');
+
+  dvv.config({
+    staticPath: '/../client',
+    timeout: 25000,
+    data: dataArray,
+    width: width,
+    height: height,
+    func: 'math.inv',
+    clock: true,
+    callback: function(results) { return results; }
+  });
 
-// //Test : invert a matrix using math.js's math.inv function imported on client side
-// //partitionLength is set to 1 by default
-dvv.config({
- staticPath: '/../client',
- timeout: 25000,
- data: createMatrixArrays(200, 5),
- func: 'math.inv',
- clock: true,
- callback: function(results) { return results; }
-});
+  dvv.start();
+
+};
 
-dvv.start();
\ No newline at end of file
+imageUtils.createDataArrayFromJpegFile('server/rgb.jpg', init);

```

Comment:
@mwharris333 looks like the path to the image is wrong. Try using `imageUtils.createDataArrayFromJpegFile(__dirname + '/assets/rgb.jpg', init);`